google/cloud/storage/oauth2/google_credentials_test.cc

```diff
@@ -97,7 +97,9 @@ TEST_F(GoogleCredentialsTest, LoadValidAuthorizedUserCredentialsViaEnvVar) {
   // Test that the authorized user credentials are loaded as the default when
   // specified via the well known environment variable.
   SetEnv(GoogleAdcEnvVar(), filename.c_str());
-  auto credentials = GoogleDefaultCredentials();
+  auto status_or_creds = GoogleDefaultCredentials();
```

Comment:
Here and elsewhere in the file, I think the preference would be to name these variables simply `credentials`.